builder/googlecompute/step_create_instance.go

```diff
@@ -126,8 +126,12 @@ func (s *StepCreateInstance) Run(ctx context.Context, state multistep.StateBag)
 		return multistep.ActionHalt
 	}
 
-	// Store source image name for use in PARtifact.
-	s.GeneratedData.Put("SourceImageName", sourceImage.Name)
+	// The export post processor uses the `StepCreateInstance`, but with this we
+	// avoid overwritting the SourceImageName.
```

Comment:
That makes sense! 
I think the panic could also be fixed if the post-processor passes the generated data it receives as an argument. But we would need to check if that's necessary 🤔

Reply:
Yeah, but in the case the export happens after a builder, we will lose the actual source image data as it will be overwritten.

Reply:
Merging now to at least fix the panic, please tell me @nywilken if you think this should be different !

Reply:
Oh nice catch. I think this is all we need especially for the Google export post-processors as the source image name that gets generated is not the name of the exported source, but of the temp instance that is created to run the export process. So in terms of the export pps this value is useless. 

I think this is what you are saying with your comment "but with this we avoid overwritting the SourceImageName", right? 



I think the correct source image